src/xcode/ENA/.swiftlint.yml

```diff
@@ -208,7 +208,7 @@ included:
 
 # Excluded paths (takes precedence over 'included')
 excluded: # paths to ignore during linting. Takes precedence over `included`.
-
+  - ENA/Backend # generated files
```

Comment:
Oh yeah… no linting of generated files. This should normally happen but I think it's ok to just ignore this rather than tweak the code generation